guides/source/active_model_basics.md

```diff
@@ -185,7 +185,7 @@ irb> person.changes
 => {"first_name"=>[nil, "First Name"]}
 ```
 
-#### Attribute based accessor methods
+#### Attribute-based accessor methods
```

Comment:
Could you create a separate PR just fixing all "x based" constructs where it should be "x-based.
For example, I think the following should probably be "string-based" instead.
https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/actionpack/lib/action_dispatch/middleware/cookies.rb#L106

Reply:
@p8 New x-based PR is here, [42485](https://github.com/rails/rails/pull/42485)
Have updated this PR as well, or are we dumping this entire PR ?